packages/storybook/stories/components/card/card.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import { type Meta, type StoryObj } from '@storybook/react';
+import React from 'react';
+import { OdsCard, type OdsCardProp } from '../../../../ods-react/src/components/card/src';
+import { CONTROL_CATEGORY } from '../../../src/constants/controls';
+import { orderControls } from '../../../src/helpers/controls';
+
+type Story = StoryObj<OdsCardProp>;
+
+const meta: Meta<OdsCardProp> = {
+  component: OdsCard,
+  // @ts-ignore see https://github.com/storybookjs/storybook/issues/27535
+  // subcomponents: { OdsCardXxx }, // Uncomment if sub components, otherwise remove
+  title: 'ODS Components/Card',
+};
+
+export default meta;
+
+export const Demo: Story = {
+  argTypes: orderControls({
+    color: {
+      table: {
+        category: CONTROL_CATEGORY.design,
+        defaultValue: { summary: 'primary' },
+        type: { summary: 'string' }
+      },
+      control: 'select',
+      options: ['primary', 'neutral'],
+    },
+    children: {
+      table: {
+        category: CONTROL_CATEGORY.slot,
+        defaultValue: { summary: 'ø' },
+      },
+      control: 'text',
+    },
+  }),
+  args: {
+    color: 'primary',
```

Comment:
TBD probably not necessary, as it's the default value already
Removing it means no value selected on select control, which could be:
- ok, to show default behaviour if nothing set
- ko? as changing select to primary do nothing as it is already the current value

BTW other components controls are already not defined, so we need to change everywhere if we change our way.